dataactcore/scripts/raw_sql/fileE.sql

```diff
@@ -0,0 +1,32 @@
+WITH submission_duns AS
+    (SELECT awardee_or_recipient_uniqu
+    FROM (
+        SELECT DISTINCT awardee_or_recipient_uniqu
+        FROM award_procurement
+        WHERE submission_id = {0}
+        UNION
+        SELECT DISTINCT awardee_or_recipient_uniqu
+        FROM award_financial_assistance
+        WHERE submission_id = {0}) AS temp)
+SELECT
+    DISTINCT ON (awardee_or_recipient_uniqu)
```

Comment:
Can this pull the latest DUNS by activation date? Just making sure they have the latest DUNS we have.

Reply:
Ordered by duns_id as discussed in person.